__tests__/topics.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+//
+test('404: Responds with "Not Found" for an invalid endpoint', () => {
+  return request(app)
+    .get("/api/i-am-very-wrong")
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.message).toBe("Not found");
+    });
+});
```

Comment:
It would be good to also give this test a describe block so it's clearly sectioned from your other tests